File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/user/exception/EmptyPhoneNumException.java
package band.gosrock.domain.domains.user.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class EmptyPhoneNumException extends DuDoongCodeException {
    public static final DuDoongCodeException EXCEPTION = new EmptyPhoneNumException();

    private EmptyPhoneNumException() {
        super(UserErrorCode.USER_PHONE_EMPTY);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/user/exception/EmptyPhoneNumException.kt
package band.gosrock.domain.domains.user.exception

import band.gosrock.common.exception.DuDoongCodeException

class EmptyPhoneNumException private constructor() : DuDoongCodeException(UserErrorCode.USER_PHONE_EMPTY) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = EmptyPhoneNumException()
    }
}
